lib/mix/lib/mix/project.ex

```diff
@@ -67,9 +67,15 @@ defmodule Mix.Project do
     * `:preferred_cli_env` - a keyword list of `{task, env}` tuples where `task`
       is the task name as an atom (for example, `:"deps.get"`) and `env` is the
       preferred environment (for example, `:test`). This option overrides what
-      specified by the tasks with the `@preferred_cli_env` attribute (see the
+      is specified by the tasks with the `@preferred_cli_env` attribute (see the
       docs for `Mix.Task`). Defaults to `[]`.
 
+    * `:preferred_cli_target` - a keyword list of `{task, target}` tuples where
+      `task` is the task name as an atom (for example, `:"test"`) and `target`
+      is the preferred target (for example, `:host`). This option overrides what
```

Comment:
I think the sentence "This option overrides what ..." should be removed since the support for the `@preferred_cli_target` attribute was dropped for now 🙂